src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/StandardOleMarshalObject.cs

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32;
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public class StandardOleMarshalObject : MarshalByRefObject, IMarshal
+    {
+        private static readonly Guid Clsid_StdMarshal = new Guid("00000017-0000-0000-c000-000000000046");
```

Comment:
This should be `CLSID_StdMarshal`. https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md#naming